### ISSUE TITLE
Allow set description for Operands

### ISSUE THREAD

**Issue by ycgambo** [getopt-php/getopt-php#125]
just as Options did.

**Comment by tflori** [comment]
To add a description to operands is trivial. But the output in the default help class opens some questions.

I'm sure not everyone prefers to have this added to the output (as long as they don't describe the operands):
```console
$ cp
Operand source is required
Usage: cp <source> [<target>]

Copies <source> to <target> or the current directory.

Operands:
  <source>  
  [<target>]
```

Maybe to hide the operands list when no operand has a description, or just show it by an extra option:
```php
$getOpt = new GetOpt();
// configure...
$getOpt->getHelpText(['showOperands' => true]);
```

**Comment by ycgambo** [comment]
An extra option is not necessary. 
One only add these Operands because it's necessary to guide their users.
So I think maybe code the detection is better. 

Usually, Operands are described in the long description
```
SCP(1)                    BSD General Commands Manual                   SCP(1)

NAME
     scp -- secure copy (remote file copy program)

SYNOPSIS
     scp [-12346BCpqrv] [-c cipher] [-F ssh_config] [-i identity_file] [-l limit]
         [-o ssh_option] [-P port] [-S program] [[user@]host1:]file1 ...
         [[user@]host2:]file2

DESCRIPTION
     scp copies files between hosts on a network.  It uses ssh(1) for data transfer, and
     uses the same authentication and provides the same security as ssh(1).  scp will ask
     for passwords or passphrases if they are needed for authentication.

     File names may contain a user and host specification to indicate that the file is to
     be copied to/from that host.  Local file names can be made explicit using absolute or
     relative pathnames to avoid scp treating file names containing `:' as host speci-
     fiers.  Copies between two remote hosts are also permitted.
```
In your package, Multi-Operands are allowed which shows a balance that to use Operands to read required options.  

This is what I did. And I naturally to call setDescription on Operands but failed.

In this kind of view, Operands and Options are same important for a package user.



On the other side, Short description is more neat than a long description. If your package user want to build a big application, he will definitely want to write a short description other than a man page for those confusing Operands.

Sadly, A PHP application is different from cli commands, we can't show a man page.

**Comment by ycgambo** [comment]
Most application use Options because they have default value built in.

But remote IP and port and several others like token are hard to set the default value and may have confusing format.

I can see that this package want to be suitable on many Options. 

Thats why I think add a description to operands is needed.

**Comment by ycgambo** [comment]
alternatively, show the Operand and it's help message only that Operand has been set a description, if not, hide both

**Comment by tflori** [comment]
@ycgambo maybe you can give `dev-feature-operand-descriptions` a try if this will solve your problem.

**Comment by tflori** [comment]
solved with #129
